.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ orbs:
 references:
   container_config: &container_config
     docker:
-      - image: circleci/node
+      - image: circleci/node:10
```

Comment:
We have a fix in for this that is in review right now. Should be looked at this morning